docs/map.json

```diff
@@ -670,6 +670,11 @@
             "name": "lint executor",
             "id": "lint",
             "file": "angular/api-linter/executors/lint"
+          },
+          {
+            "name": "workspace-rule generator",
+            "id": "lint",
```

Comment:
You need to change the ID of this to something unique. Currently, it is linking to the `lint` executor.